pkg/addon/templateagent/values.go

```diff
@@ -87,7 +96,7 @@ func (a *CRDTemplateAgentAddon) getValues(
 	if err != nil {
 		return presetValues, overrideValues, privateValues, nil
 	}
-	overrideValues = addonfactory.MergeValues(overrideValues, builtinValues)
+	overrideValues = addonfactory.MergeValues(builtinValues, overrideValues)
```

Comment:
the builtin Values are immutable

Reply:
I thought we want overridevalue to override builtinValue?

Reply:
we want the overridevalue to override defaultValue, but for bultinValues(ClusterName), they should be immutable.

Reply:
I am a bit confusing, I thought this func is return overrideValues that is using builtingValues as base and override by overrideValue if the key exists. The buitInvalue is immutable as input.